apps/remote_control/lib/lexical/remote_control/code_intelligence/entity.ex

```diff
@@ -12,12 +12,13 @@ defmodule Lexical.RemoteControl.CodeIntelligence.Entity do
   require Logger
   require Sourceror.Identifier
 
+  @type maybe_module :: module() | nil
   @type resolved ::
-          {:module, module()}
-          | {:struct, module()}
-          | {:call, module(), fun_name :: atom(), arity :: non_neg_integer()}
-          | {:type, module(), type_name :: atom(), arity :: non_neg_integer()}
-          | {:module_attribute, container_module :: module(), attribut_name :: atom()}
+          {:module, maybe_module()}
+          | {:struct, maybe_module()}
+          | {:call, maybe_module(), fun_name :: atom(), arity :: non_neg_integer()}
+          | {:type, maybe_module(), type_name :: atom(), arity :: non_neg_integer()}
+          | {:module_attribute, container_module :: maybe_module(), attribut_name :: atom()}
```

Comment:
Just noticed this small typo.

```suggestion
          | {:module_attribute, container_module :: maybe_module(), attribute_name :: atom()}
```